cmd/mt-fakemetrics/policy/policy.go

```diff
@@ -0,0 +1,109 @@
+package policy
+
+import (
+	"fmt"
+	"math/rand"
+	"strconv"
+	"strings"
+)
+
+type ValuePolicy interface {
+	Value(ts int64) float64
+}
+
+type ValuePolicyNone struct {
```

Comment:
Is this called `*None` because it's used when no value policy has been specified? I think it would make more sense to name it something like `ValuePolicyRandom` because that's what it actually is, and then still use it as the default if no policy is specified

Reply:
Yes, that makes sense

Reply:
+1